packages/components/src/tabs/index.tsx

```diff
@@ -36,11 +36,14 @@ function internalToExternalTabId(
 }
 
 /**
- * Display one panel of content at a time with a tabbed interface, based on the
- * WAI-ARIA Tabs Pattern⁠.
+ * Tabs is a collection of React components that combine to render
+ * an [ARIA-compliant tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).
  *
- * @see https://www.w3.org/WAI/ARIA/apg/patterns/tabs/
- * ```
+ * Tabs organizes content across different screens, data sets, and interactions.
+ * It has two sections: a list of tabs, and the view to show when tabs are chosen.
```

Comment:
Only one tab can be chosen at a given time, so I think it makes sense to reflect that more accurately. 

```suggestion
 * It has two sections: a list of tabs, and the view to show when a tab is chosen.
```